src/services/TwitterService.ts

```diff
@@ -31,12 +31,14 @@ class TwitterService {
 		}).on("data", listener => this.handleTwitterStream(listener, tweetChannel));
 	}
 
-	handleTwitterStream = async ({ id_str: id, text, extended_tweet }: TwitterStreamListener, tweetChannel: TextChannel): Promise<void> => {
+	handleTwitterStream = async ({ id_str: id, text, user, retweeted_status, extended_tweet }: TwitterStreamListener, tweetChannel: TextChannel): Promise<void> => {
```

Comment:
This line is starting to get rather long, can we do the destructing on a new line, i.e.:
```ts
handleTwitterStream = async (stream: TwitterStreamListener, tweetChannel: TextChannel): Promise<Void> => {
    const { id_str: id, text, user, retweeted_status, extended_tweet} = stream;
    ...
}
```